bika/lims/content/bikasetup.py

```diff
@@ -319,7 +368,7 @@ class PrefixesField(RecordsField):
     ),
     BooleanField(
         'SamplingWorkflowEnabled',
-        schemata="Analyses",
+        schemata="Sampling and COC",
```

Comment:
Please leave this Schemata as `Analyses`, because we won't integrate COC at the moment

Reply:
@ramonski a small question here. All the Schematas specified as ```Sampling and COC``` should be replaced by ```Analyses``` or just the one you mention?

Reply:
@juangallostra: Please change the "Sampling and COC" to "Analysis" for the whole file, because we currently do not have COC in place.
However, I don't know how we proceed with https://github.com/bikalims/bika.lims/pull/2112 (see also Gitter channel). But it is easy enough to change that later again.